packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -53,62 +53,43 @@ export interface ViewEndedEvent {
 export const THROTTLE_VIEW_UPDATE_PERIOD = 3000
 export const SESSION_KEEP_ALIVE_INTERVAL = 5 * ONE_MINUTE
 
-export function trackViews(location: Location, lifeCycle: LifeCycle) {
+type View = ReturnType<typeof newView>
+
+export function trackViews(location: Location, lifeCycle: LifeCycle, areViewsTrackedAutomatically: boolean) {
   let isRecording = false
+  let hasInitialView = false
+  let currentView: View
+  let stopInitialViewTracking: () => void
+  let stopViewLifeCycle: () => void
+  let stopLocationChangesTracking: (() => void) | undefined
```

Comment:
We might want to explicit this as a state like:

```ts
type TrackViewsState =
{
  status: TrackViewStatus.WaitingForInitialView,
} |
{
  status: TrackViewStatus.Started,
  currentView: View,
  stop: () => void,
} |
{
  status: TrackViewStatus.Stopped
}
```

It would make clear that `currentView` is not always defined, and what should be called to stop the view tracking.

Reply:
I've updated the strategy to always have an initial view